lib/travis/build/script.rb

```diff
@@ -45,6 +45,10 @@ module Travis
   module Build
     class Script
       TEMPLATES_PATH = File.expand_path('../templates', __FILE__)
+      INTERNAL_RUBY_REGEX = ENV.fetch(
+        'TRAVIS_INTERNAL_RUBY_REGEX',
+        'ruby-(2\.[0-2]\.[0-9]|1\.9\.3)'
```

Comment:
@henrikhodne We are lazily skipping `jruby-*` here (assuming that JRuby all of a sudden start using these version numbers).